2024-10-06 21:43:43,809 INFO    StreamThr :8690 [internal.py:wandb_internal():85] W&B internal server running at pid: 8690, started at: 2024-10-06 21:43:43.809044
2024-10-06 21:43:43,810 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: status
2024-10-06 21:43:43,812 INFO    WriterThread:8690 [datastore.py:open_for_write():87] open: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/run-zdfimv4a.wandb
2024-10-06 21:43:43,812 DEBUG   SenderThread:8690 [sender.py:send():379] send: header
2024-10-06 21:43:43,812 DEBUG   SenderThread:8690 [sender.py:send():379] send: run
2024-10-06 21:43:44,000 INFO    SenderThread:8690 [dir_watcher.py:__init__():211] watching files in: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files
2024-10-06 21:43:44,001 INFO    SenderThread:8690 [sender.py:_start_run_threads():1188] run started: zdfimv4a with start time 1728269023.809782
2024-10-06 21:43:44,006 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: check_version
2024-10-06 21:43:44,006 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: check_version
2024-10-06 21:43:44,062 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: run_start
2024-10-06 21:43:44,069 DEBUG   HandlerThread:8690 [system_info.py:__init__():26] System info init
2024-10-06 21:43:44,069 DEBUG   HandlerThread:8690 [system_info.py:__init__():41] System info init done
2024-10-06 21:43:44,069 INFO    HandlerThread:8690 [system_monitor.py:start():194] Starting system monitor
2024-10-06 21:43:44,069 INFO    SystemMonitor:8690 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-10-06 21:43:44,070 INFO    HandlerThread:8690 [system_monitor.py:probe():214] Collecting system info
2024-10-06 21:43:44,072 INFO    SystemMonitor:8690 [interfaces.py:start():188] Started cpu monitoring
2024-10-06 21:43:44,072 INFO    SystemMonitor:8690 [interfaces.py:start():188] Started disk monitoring
2024-10-06 21:43:44,074 ERROR   HandlerThread:8690 [gpu_apple.py:probe():176] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:44,074 INFO    SystemMonitor:8690 [interfaces.py:start():188] Started gpuapple monitoring
2024-10-06 21:43:44,076 ERROR   gpuapple  :8690 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:44,078 DEBUG   HandlerThread:8690 [system_info.py:probe():152] Probing system
2024-10-06 21:43:44,079 INFO    SystemMonitor:8690 [interfaces.py:start():188] Started memory monitoring
2024-10-06 21:43:44,080 INFO    SystemMonitor:8690 [interfaces.py:start():188] Started network monitoring
2024-10-06 21:43:44,082 DEBUG   HandlerThread:8690 [gitlib.py:_init_repo():56] git repository is invalid
2024-10-06 21:43:44,082 DEBUG   HandlerThread:8690 [system_info.py:probe():200] Probing system done
2024-10-06 21:43:44,082 DEBUG   HandlerThread:8690 [system_monitor.py:probe():223] {'os': 'macOS-14.6.1-arm64-arm-64bit', 'python': '3.12.2', 'heartbeatAt': '2024-10-07T02:43:44.079754', 'startedAt': '2024-10-07T02:43:43.802964', 'docker': None, 'cuda': None, 'args': ('data.pkl',), 'state': 'running', 'program': '/Users/milind/Desktop/valence/milind_code/non_llm.py', 'codePathLocal': 'non_llm.py', 'codePath': 'non_llm.py', 'host': 'Milinds-MacBook-Pro.local', 'username': 'milind', 'executable': '/opt/anaconda3/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 228.27386474609375, 'used': 166.7853012084961}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-10-06 21:43:44,082 INFO    HandlerThread:8690 [system_monitor.py:probe():224] Finished collecting system info
2024-10-06 21:43:44,082 INFO    HandlerThread:8690 [system_monitor.py:probe():227] Publishing system info
2024-10-06 21:43:44,082 DEBUG   HandlerThread:8690 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-10-06 21:43:45,006 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/conda-environment.yaml
2024-10-06 21:43:46,086 ERROR   gpuapple  :8690 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:46,132 DEBUG   HandlerThread:8690 [system_info.py:_save_conda():224] Saving conda packages done
2024-10-06 21:43:46,133 INFO    HandlerThread:8690 [system_monitor.py:probe():229] Finished publishing system info
2024-10-06 21:43:46,135 DEBUG   SenderThread:8690 [sender.py:send():379] send: files
2024-10-06 21:43:46,136 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-10-06 21:43:46,366 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: python_packages
2024-10-06 21:43:46,366 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: python_packages
2024-10-06 21:43:46,367 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: stop_status
2024-10-06 21:43:46,367 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:46,367 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: stop_status
2024-10-06 21:43:46,382 INFO    wandb-upload_0:8690 [upload_job.py:push():130] Uploaded file /var/folders/jv/wvjkll0d0vl_clbchr37hpk40000gn/T/tmpfnz_6vjhwandb/iogcbypu-wandb-metadata.json
2024-10-06 21:43:46,429 DEBUG   SenderThread:8690 [sender.py:send():379] send: telemetry
2024-10-06 21:43:47,016 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/conda-environment.yaml
2024-10-06 21:43:47,017 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/requirements.txt
2024-10-06 21:43:47,017 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/output.log
2024-10-06 21:43:47,017 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/wandb-metadata.json
2024-10-06 21:43:47,372 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:48,091 ERROR   gpuapple  :8690 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:48,376 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:49,028 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/output.log
2024-10-06 21:43:49,398 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:49,452 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:43:50,110 ERROR   gpuapple  :8690 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:50,401 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:51,388 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:52,122 ERROR   gpuapple  :8690 [gpu_apple.py:sample():110] GPU stats error: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/anaconda3/lib/python3.12/subprocess.py", line 1953, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/anaconda3/lib/python3.12/site-packages/wandb/bin/apple_gpu_stats'
2024-10-06 21:43:52,406 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:53,315 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,315 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,316 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,316 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,316 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,316 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,317 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,317 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,317 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,317 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,318 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,318 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,318 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,318 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,319 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,319 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,319 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,320 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,320 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,320 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,320 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,320 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,321 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,321 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,321 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,321 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,321 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,322 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,322 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,322 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,429 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:53,941 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: partial_history
2024-10-06 21:43:53,941 DEBUG   SenderThread:8690 [sender.py:send():379] send: history
2024-10-06 21:43:53,942 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: summary_record
2024-10-06 21:43:53,942 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,945 DEBUG   SenderThread:8690 [sender.py:send():379] send: telemetry
2024-10-06 21:43:53,946 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: server_info
2024-10-06 21:43:53,946 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: server_info
2024-10-06 21:43:53,987 DEBUG   SenderThread:8690 [sender.py:send():379] send: exit
2024-10-06 21:43:53,987 INFO    SenderThread:8690 [sender.py:send_exit():586] handling exit code: 0
2024-10-06 21:43:53,987 INFO    SenderThread:8690 [sender.py:send_exit():588] handling runtime: 9
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:send_exit():594] send defer
2024-10-06 21:43:53,988 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 0
2024-10-06 21:43:53,988 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 0
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 1
2024-10-06 21:43:53,988 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 1
2024-10-06 21:43:53,988 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 1
2024-10-06 21:43:53,988 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 2
2024-10-06 21:43:53,988 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 2
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [system_monitor.py:finish():203] Stopping system monitor
2024-10-06 21:43:53,988 DEBUG   SystemMonitor:8690 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-10-06 21:43:53,988 DEBUG   SystemMonitor:8690 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [interfaces.py:finish():200] Joined cpu monitor
2024-10-06 21:43:53,988 DEBUG   SystemMonitor:8690 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-10-06 21:43:53,988 INFO    HandlerThread:8690 [interfaces.py:finish():200] Joined disk monitor
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [interfaces.py:finish():200] Joined gpuapple monitor
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [interfaces.py:finish():200] Joined memory monitor
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [interfaces.py:finish():200] Joined network monitor
2024-10-06 21:43:53,989 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 2
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 3
2024-10-06 21:43:53,989 DEBUG   SenderThread:8690 [sender.py:send():379] send: telemetry
2024-10-06 21:43:53,989 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,989 DEBUG   SenderThread:8690 [sender.py:send():379] send: stats
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 3
2024-10-06 21:43:53,989 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 3
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 4
2024-10-06 21:43:53,989 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 4
2024-10-06 21:43:53,989 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 4
2024-10-06 21:43:53,989 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 5
2024-10-06 21:43:53,989 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,989 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 5
2024-10-06 21:43:53,990 DEBUG   SenderThread:8690 [sender.py:send():379] send: summary
2024-10-06 21:43:53,990 INFO    SenderThread:8690 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-10-06 21:43:53,990 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,990 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 5
2024-10-06 21:43:53,990 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 6
2024-10-06 21:43:53,990 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:53,990 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 6
2024-10-06 21:43:53,990 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:53,990 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 6
2024-10-06 21:43:53,991 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: status_report
2024-10-06 21:43:54,056 INFO    Thread-12 :8690 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/wandb-summary.json
2024-10-06 21:43:54,095 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 7
2024-10-06 21:43:54,095 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:54,095 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 7
2024-10-06 21:43:54,095 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:54,095 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 7
2024-10-06 21:43:54,491 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 8
2024-10-06 21:43:54,492 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:54,492 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 8
2024-10-06 21:43:54,492 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:54,492 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 8
2024-10-06 21:43:54,493 INFO    SenderThread:8690 [job_builder.py:build():440] Attempting to build job artifact
2024-10-06 21:43:54,494 INFO    SenderThread:8690 [job_builder.py:_get_source_type():580] no source found
2024-10-06 21:43:54,494 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 9
2024-10-06 21:43:54,494 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:54,494 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 9
2024-10-06 21:43:54,494 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:54,494 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 9
2024-10-06 21:43:54,494 INFO    SenderThread:8690 [dir_watcher.py:finish():358] shutting down directory watcher
2024-10-06 21:43:54,993 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 21:43:55,062 INFO    SenderThread:8690 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/config.yaml
2024-10-06 21:43:55,063 INFO    SenderThread:8690 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/output.log
2024-10-06 21:43:55,064 INFO    SenderThread:8690 [dir_watcher.py:finish():388] scan: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files
2024-10-06 21:43:55,064 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/requirements.txt requirements.txt
2024-10-06 21:43:55,064 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/output.log output.log
2024-10-06 21:43:55,067 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/config.yaml config.yaml
2024-10-06 21:43:55,069 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/wandb-summary.json wandb-summary.json
2024-10-06 21:43:55,070 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/conda-environment.yaml conda-environment.yaml
2024-10-06 21:43:55,072 INFO    SenderThread:8690 [dir_watcher.py:finish():402] scan save: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/wandb-metadata.json wandb-metadata.json
2024-10-06 21:43:55,073 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 10
2024-10-06 21:43:55,073 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 21:43:55,075 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:55,075 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 10
2024-10-06 21:43:55,075 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:55,076 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 10
2024-10-06 21:43:55,076 INFO    SenderThread:8690 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 21:43:55,270 INFO    wandb-upload_0:8690 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/requirements.txt
2024-10-06 21:43:55,312 INFO    wandb-upload_3:8690 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/wandb-summary.json
2024-10-06 21:43:55,329 INFO    wandb-upload_1:8690 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/output.log
2024-10-06 21:43:55,329 INFO    wandb-upload_4:8690 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/conda-environment.yaml
2024-10-06 21:43:55,369 INFO    wandb-upload_2:8690 [upload_job.py:push():130] Uploaded file /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/files/config.yaml
2024-10-06 21:43:55,574 INFO    Thread-11 (_thread_body):8690 [sender.py:transition_state():614] send defer: 11
2024-10-06 21:43:55,575 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:55,575 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 11
2024-10-06 21:43:55,576 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:55,576 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 11
2024-10-06 21:43:55,576 INFO    SenderThread:8690 [file_pusher.py:join():175] waiting for file pusher
2024-10-06 21:43:55,578 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 12
2024-10-06 21:43:55,578 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:55,579 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 12
2024-10-06 21:43:55,579 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:55,579 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 12
2024-10-06 21:43:55,579 INFO    SenderThread:8690 [file_stream.py:finish():601] file stream finish called
2024-10-06 21:43:55,659 INFO    SenderThread:8690 [file_stream.py:finish():605] file stream finish is done
2024-10-06 21:43:55,659 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 13
2024-10-06 21:43:55,660 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:55,660 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 13
2024-10-06 21:43:55,660 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:55,660 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 13
2024-10-06 21:43:55,660 INFO    SenderThread:8690 [sender.py:transition_state():614] send defer: 14
2024-10-06 21:43:55,661 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: defer
2024-10-06 21:43:55,661 INFO    HandlerThread:8690 [handler.py:handle_request_defer():184] handle defer: 14
2024-10-06 21:43:55,661 DEBUG   SenderThread:8690 [sender.py:send():379] send: final
2024-10-06 21:43:55,661 DEBUG   SenderThread:8690 [sender.py:send():379] send: footer
2024-10-06 21:43:55,661 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: defer
2024-10-06 21:43:55,661 INFO    SenderThread:8690 [sender.py:send_request_defer():610] handle sender defer: 14
2024-10-06 21:43:55,663 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: poll_exit
2024-10-06 21:43:55,663 DEBUG   SenderThread:8690 [sender.py:send_request():406] send_request: poll_exit
2024-10-06 21:43:55,665 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-06 21:43:55,665 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: get_summary
2024-10-06 21:43:55,666 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: sampled_history
2024-10-06 21:43:55,669 DEBUG   HandlerThread:8690 [handler.py:handle_request():158] handle_request: shutdown
2024-10-06 21:43:55,669 INFO    HandlerThread:8690 [handler.py:finish():882] shutting down handler
2024-10-06 21:43:56,667 INFO    WriterThread:8690 [datastore.py:close():296] close: /Users/milind/Desktop/valence/milind_code/wandb/run-20241006_214343-zdfimv4a/run-zdfimv4a.wandb
2024-10-06 21:43:56,668 INFO    SenderThread:8690 [sender.py:finish():1608] shutting down sender
2024-10-06 21:43:56,668 INFO    SenderThread:8690 [file_pusher.py:finish():169] shutting down file pusher
2024-10-06 21:43:56,668 INFO    SenderThread:8690 [file_pusher.py:join():175] waiting for file pusher
